src/styles/colors.tsx

```diff
@@ -39,6 +39,9 @@ export enum Colors {
   successDark = '#137211',
   successLight = '#F1FDF1',
 
+  informationDark = '#0768AE',
+  informationFaint = '#E8F8FF',
```

Comment:
for this color i'd just go with `onboardingLightBlue`

Reply:
I think it's better to try to reuse the colors if possible, although I think informationFaint is easier to see/read in this case. I'll keep informationFaint.

| Colors.informationFaint | Colors.onboardingLightBlue
| -- | -- |
| ![image](https://github.com/valora-inc/wallet/assets/147835134/093403e9-4c58-474b-a14a-4d5d29bcda26) | ![image](https://github.com/valora-inc/wallet/assets/147835134/a2a7db94-211b-464c-a809-5ead09b24169) |

Reply:
i think my main point here is that we are trying to make our design system in code the same as what exists in Figma since we have RET-913 coming up. i suggest using an existing color if it does no harm, but if you think that it's not a good alternative then we should check with Nitya if this color belongs in the design system and align on the name she will call it in Figma going forward (i think on last inspection, i did not see a name for it).

Reply:
The [Figma design](https://www.figma.com/file/erFfzHvSTm5g1sjK6jWyEH/Working-Design-System-Components?type=design&node-id=213-5&mode=design&t=oTFPVxN7Czm77Rb3-4) for Typography & Color does have `#E8F8FF` color, it's called Info-Light.

I've mentioned to Nitya that we don't have the Info-Light color, but we have `onboardingLightBlue` color `#D6EAF3`. She has updated the design, which references Info-Light color and mentioned I can this new color to the app.

To keep it consistent, I've renamed it to `infoLight` in the code (and also renamed the other color to `infoDark`, it was used in only one place).